dashboard/management/commands/createuser.py

```diff
@@ -0,0 +1,31 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User
+import random, string
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('--username', dest='username', type=str, required=True)
+        parser.add_argument('--email', dest='email', type=str, required=True)
+        parser.add_argument('--password', dest='password', type=str, required=False)
+        parser.add_argument('--superuser', dest='superuser', action='store_true', required=False)
+
+    def handle(self, *args, **options):
+        username = options.get('username')
+        password = options.get('password')
+        password = ''.join(random.sample(string.ascii_letters, 8)) if password is None else password
```

Comment:
8 because of Uniqname character limit is 8?

Reply:
This is an 8 character password that I just created for development. In production it will use SAML so this is irrelevant. This could actually just probably have been a hardcoded password or blank? I could make this a constant in this file I guess. ;) You can specify the password in the --password string too.